core/src/main/java/org/apache/accumulo/core/client/mapred/AbstractInputFormat.java

```diff
@@ -397,15 +414,17 @@ public static InputTableConfig getInputTableConfig(JobConf job, String tableName
      *          the table name for which the scanner is configured
      * @since 1.6.0
      */
-    protected abstract void setupIterators(JobConf job, Scanner scanner, String tableName, RangeInputSplit split);
+    @Deprecated
```

Comment:
Another common practice in the javadoc is to provide a pointer to the replacement for the deprecated method if one exists.